docs/java-rest/high-level/security/invalidate-token.asciidoc

```diff
@@ -0,0 +1,72 @@
+[[java-rest-high-security-invalidate-token]]
```

Comment:
Can you switch this to the new dry docs format? Baz has been asking new PRs to use that way. Clear roles cache should be an example for security